.github/workflows/release.yml

```diff
@@ -36,17 +36,20 @@ jobs:
           poetry run semantic-release version --no-changelog
 
           NEXT_VERSION=$(poetry run semantic-release --noop version --print-tag)
-          echo "Next version: ${NEXT_VERSION}"
+          NEXT_VERSION_NO_PREFIX=${NEXT_VERSION#v}
+          echo "Next version: ${NEXT_VERSION_NO_PREFIX}"
```

Comment:
The version from `semantic-release` looks like this:  `v0.6.0`

The version from PyPI looks like this:  `0.6.0`

This is why the `v` is removed before making the comparison.